include/field_cache.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <string>
+#include <map>
+#include <stack>
+
+namespace quda {
```

Comment:
THis file needs some doxygen

Reply:
[b37ed336](https://github.com/lattice/quda/commit/b37ed336c69a82e851d2cfccd3ca5472c750f718)